metrics-dropwizard/README.md

```diff
@@ -0,0 +1,140 @@
+# Metrics
+
+This is a [Dropwizard Metrics](https://github.com/dropwizard/metrics) implementation 
+of the USI metrics interface. It supports multiple exporters ([DataDog(https://www.datadoghq.com/)],
+[StatsD](https://github.com/etsy/statsd)) as well as text representation in 
+[Prometheus](https://prometheus.io/docs/instrumenting/exposition_formats/) format.
+
+Histograms and timers are backed with reservoirs leveraging
+[HdrHistogram](http://hdrhistogram.org/).
+
+## Metric names
+
+All metric names are prefixed with `usi` by default. This can be changed
+using `usi.metrics.name-prefix` configuration parameter. Metric name components 
+are joined with dots. Components may have dashes in them.
+
+A metric type and a unit of measurement (if any) are appended to
+a metric name. A couple of examples:
+
+* `usi.scheduler.offers.counter`
```

Comment:
this would be a blocker for me if we didn't have a way to configure the `name-prefix`.   At this point it isn't a blocker but I would like to make the ability to use USI impossible without initializing the scheduler name etc.